android/app/src/main/kotlin/org/getlantern/lantern/MainActivity.kt

```diff
@@ -274,25 +287,33 @@ class MainActivity :
                     accountInitDialog?.dismiss()
                     finish()
                 }
-                accountInitDialog?.show()
+                if (!isFinishing) {
+                    accountInitDialog?.show()
+                }
             }
 
             AccountInitializationStatus.Status.SUCCESS -> {
-                accountInitDialog?.let { it.dismiss() }
+                if (!isFinishing) {
+                    accountInitDialog?.dismiss()
+                }
```

Comment:
Thanks for making these changes! Just curious, do we need this account initialization process anymore? On my related PR, I removed the step setting this to "initializing" after Lantern starts. Now for fresh installs, the dialog no longer appears, and Lantern works seamlessly, leading to an overall better experience when the app is first opened

Reply:
Once we merge your changes, we do not need it.